node/pkg/processor/message.go

```diff
@@ -105,7 +107,11 @@ func (p *Processor) handleMessage(ctx context.Context, k *common.MessagePublicat
 			panic("failed to unmarshal VAA from db")
 		}
 
-		if k.Timestamp.Sub(existing.Timestamp) > settlementTime {
+		if time.Now().Sub(existing.Timestamp) > settlementTime &&
```

Comment:
Could a guardian mess with things if they were to change their system time? Also, unlikely, just trying to consider all of the ways this could be used.